HedgeLib/Sets/SOBJ.cs

```diff
@@ -388,60 +408,68 @@ private static void WriteObject(BINAWriter writer, SetObject obj, SOBJType type)
 
-                    foreach (uint value in arr)
-                        writer.Write(value);
+                        foreach (uint value in arr)
+                            writer.Write(value);
 
-                    continue;
-                }
-                else if (param.DataType == typeof(string))
-                {
-                    // Data Info
-                    string str = (string)param.Data;
-                    writer.AddOffset("strOffset");
-                    writer.WriteNulls(4); // TODO: Figure out what this is.
-
-                    if (string.IsNullOrEmpty(str))
+                        continue;
+                    }
+                    else if (param.DataType == typeof(string))
                     {
-                        writer.FillInOffset("strOffset", 0, true);
+                        // Data Info
+                        string str = (string)param.Data;
+                        writer.AddOffset("strOffset");
+                        writer.WriteNulls(4); // TODO: Figure out what this is.
+
+                        if (string.IsNullOrEmpty(str))
+                        {
+                            writer.FillInOffset("strOffset", 0, true);
+                        }
+                        else
+                        {
+                            writer.FillInOffset("strOffset", false);
+                            writer.WriteNullTerminatedString(str);
+                        }
+
+                        continue;
                     }
-                    else
+                    else if (param.DataType == typeof(float) ||
+                        param.DataType == typeof(int) || param.DataType == typeof(uint))
                     {
-                        writer.FillInOffset("strOffset", false);
-                        writer.WriteNullTerminatedString(str);
+                        writer.FixPadding(4);
+                    }
+                    else if (type == SOBJType.LostWorld && param.DataType == typeof(Vector3))
+                    {
+                        writer.FixPadding(16);
                     }
 
-                    continue;
-                }
-                else if (param.DataType == typeof(float) ||
-                    param.DataType == typeof(int) || param.DataType == typeof(uint))
-                {
-                    writer.FixPadding(4);
-                }
-                else if (type == SOBJType.LostWorld && param.DataType == typeof(Vector3))
-                {
-                    writer.FixPadding(16);
+                    // Write Data
+                    writer.WriteByType(param.DataType, param.Data);
                 }
-
-                // Write Data
-                writer.WriteByType(param.DataType, param.Data);
             }
-
+            else if (obj.RawParamData != null) //Objects with template but without parameters, but do have a byte length definition.
+            {
+                //Write unedited raw data retrieved from loaded orc
```

Comment:
Again, massive nit-pick here, but I'm picky with my comments lol.
Could you please add a space at the beginning of the comment after the slashes, right before the "W"?
*Sorry to be so picky, thank you*